### ISSUE TITLE
an error during install-ubuntu.sh

### ISSUE THREAD

**Issue by fjremnav** [mobile-insight/mobileinsight-core#106]
I got the following errors when I run ./install-ubuntu.sh.
Same error when I run python3 ./offline-analysis-example.py

Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/MobileInsight-5.0.0-py3.8-linux-x86_64.egg/mobile_insight/monitor/offline_replayer.py", line 209, in run
    self.send(event)
  File "/usr/local/lib/python3.8/dist-packages/MobileInsight-5.0.0-py3.8-linux-x86_64.egg/mobile_insight/element.py", line 104, in send
    self.to_list[i].recv(self, event)
  File "/usr/local/lib/python3.8/dist-packages/MobileInsight-5.0.0-py3.8-linux-x86_64.egg/mobile_insight/analyzer/analyzer.py", line 234, in recv
    self.source_callback[i](event)
  File "/usr/local/lib/python3.8/dist-packages/MobileInsight-5.0.0-py3.8-linux-x86_64.egg/mobile_insight/analyzer/lte_rrc_analyzer.py", line 249, in __rrc_filter
    self.__callback_sib_config(xml_msg)
  File "/usr/local/lib/python3.8/dist-packages/MobileInsight-5.0.0-py3.8-linux-x86_64.egg/mobile_insight/analyzer/lte_rrc_analyzer.py", line 451, in __callback_sib_config
    int(field_val['lte-rrc.utra_q_RxLevMin']) * 2,
TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'

Failed to run offline analysis example!
Exiting with status 4.

Any idea how to fix it?

Thanks,

**Comment by futuredimensions1** [comment]
Hi

I am also getting the same error.

Were you able to fix it?

**Comment by markubiak** [comment]
Seems to be an issue with the lte_rrc_analyzer, and doesn't appear to be related to the install. I was able to work around by adding the following to the for loop of line 503 on v5.0.0 branch:
```
   503                 for val in field.iter('field'):                                    
   504                     field_val[val.get('name')] = val.get('show')                   
   505                     if val.get('name') == "lte-rrc.utra_q_RxLevMin":               
   506                         field_val['lte-rrc.q_RxLevMin'] = val.get('show')
```

**Comment by VLiu7** [comment]
Thanks for your issue, I have fixed it in eee23ff418cfc25173afb1b0404a046f3421a578.
